src/Server.h

```diff
@@ -223,6 +226,7 @@ class cServer
 	AString m_FaviconData;
 	size_t m_MaxPlayers;
 	bool m_bIsHardcore;
+	bool m_RequireResourcePack;
```

Comment:
I'm not sure if this field, along with it's associated getter, is still necessary. Initially, I added it so that server owners can indicate that the resource pack is optional and we don't have to kick clients for rejecting it. But tonitch then added a call to a lua hook, so it seems like the same functionality can be done by lua?

Reply:
Actually, I think this can be a good option for non-dev to be able to enable/disable this feature. maybe we can keep both or just in plugin but then it would be better to add an option in core or web to tweak that... I don't know I'm not in the project since long enough to understand the full vision. @12xx12 @madmaxoft  What do you think ?

Reply:
I am not sure about the Lua hook. Since there is no way to resent the resource pack the only thing you could do is kicking the player and that could be done with a simple config value

Reply:
> I am not sure about the Lua hook. Since there is no way to resent the resource pack the only thing you could do is kicking the player and that could be done with a simple config value

that make sence, I just know some people think about crazy way of using plugin functionality even in java... for instance, admin would not need to accept or someting like that... but in the end... yhea it's pretty specific so might not be needed... in this case we could just reverse to https://github.com/cuberite/cuberite/commit/ba835236af3678f78c6b2fce4844907f5f536321